public/js/common/react_modal.js

```diff
@@ -0,0 +1,84 @@
+function openReactModal(courseCode) {
+    
```

Comment:
An off blank line. Also, for JavaScript functions, we tend to use `'use strict';` statements. Browse around some of the other JavaScript code, and you're likely to see a few.